deployments/container/Dockerfile

```diff
@@ -51,15 +51,15 @@ ENV NVIDIA_DRIVER_CAPABILITIES=utility
 ARG VERSION="N/A"
 ARG GIT_COMMIT="unknown"
 
-LABEL io.k8s.display-name="NVIDIA DRA Driver"
-LABEL name="NVIDIA DRA Driver"
+LABEL io.k8s.display-name="NVIDIA DRA Driver for GPUs"
+LABEL name="NVIDIA DRA Driver for GPUs"
 LABEL vendor="NVIDIA"
 LABEL version=${VERSION}
 LABEL com.nvidia.git-commit="${GIT_COMMIT}"
 LABEL release="N/A"
-LABEL summary="NVIDIA DRA driver for Kubernetes"
-LABEL description="See summary"
-LABEL org.opencontainers.image.description "NVIDIA GPU DRA driver for Kubernetes"
+LABEL summary="NVIDIA DRA Driver for GPUs"
+LABEL description="NVIDIA DRA Driver for GPUs"
+LABEL org.opencontainers.image.description "NVIDIA DRA Driver for GPUs"
```

Comment:
```suggestion
LABEL org.opencontainers.image.description="NVIDIA DRA Driver for GPUs"
```

Reply:
On Dockerfiles is valid to have Label Key Val separated  without an `=`

Reply:
> On Dockerfiles is valid to have Label Key Val separated without an =

Yes, I'm not saying it's invalid. I'm saying that it's locally inconsistent. Not a blocker for this PR though.